tests/unit/test_parameters/test_bpx.py

```diff
@@ -34,6 +34,7 @@ def setUp(self):
                     "in parallel to make a cell": 34,
                     "External surface area [m2]": 3.79e-2,
                     "Volume [m3]": 1.28e-4,
+                    "Contact resistance [Ohm]": 0,
```

Comment:
Remove this from here and the test will pass. BPX doesn't let you add extra fields that aren't in the schema, so it throws an error. You already hard-code zero contact resistance in `bpx.py`. If users want to read in BPX and add the contact parameter resistance after, they can do that in the usual way.